app/assets/stylesheets/finder_frontend.scss

```diff
@@ -407,3 +400,29 @@
   }
 }
 
+.result-region-header__counter {
+  white-space: nowrap;
+}
+
+.result-region-header__sort-label {
+  @include govuk-responsive-padding(1, "right");
+  @include govuk-font(16);
+  display: inline-block;
+  margin-bottom: govuk-spacing(3);
+}
+
+.result-region-header__sort-select {
+  @include govuk-font(16);
+  height: govuk-spacing(6);
+}
+
+.result-info {
+  margin-bottom: govuk-spacing(2);
+  .govuk-grid-column-two-thirds {
```

Comment:
Nitpick: can we put some spacing above this selector and the following mixin, to make it more readable?